corehq/apps/hqwebapp/static/registry/scss/light_color_scheme.scss

```diff
@@ -1,100 +1,100 @@
-// A paler version of Bootstraps button and label color scheme
-/*@cc-color-default-light-bg: #F5F5F6;
-@cc-color-default-light-fg: #545D6A;
-@cc-color-default-light-hover: #DADCDF;
+// A paler version of Bootstrap's button and label color scheme
+$cc-color-default-light-bg: #F5F5F6;
+$cc-color-default-light-fg: #545D6A;
+$cc-color-default-light-hover: #DADCDF;
 
-@cc-color-success-light-bg: #E3F7EE;
-@cc-color-success-light-fg: #4BC18F;
-@cc-color-success-light-hover: #CAEEDE;
+$cc-color-success-light-bg: #E3F7EE;
+$cc-color-success-light-fg: #4BC18F;
+$cc-color-success-light-hover: #CAEEDE;
 
-@cc-color-info-light-bg: #E4F5FA;
-@cc-color-info-light-fg: #5BC0DE;
-@cc-color-info-light-hover: #C8EAF4;
+$cc-color-info-light-bg: #E4F5FA;
+$cc-color-info-light-fg: #5BC0DE;
+$cc-color-info-light-hover: #C8EAF4;
 
-@cc-color-warning-light-bg: #FFF5E7;
-@cc-color-warning-light-fg: #EC9D28;
-@cc-color-warning-light-hover: #FCE6C7;
+$cc-color-warning-light-bg: #FFF5E7;
+$cc-color-warning-light-fg: #EC9D28;
+$cc-color-warning-light-hover: #FCE6C7;
 
-@cc-color-danger-light-bg: #FFEDED;
-@cc-color-danger-light-fg: #D03939;
-@cc-color-danger-light-hover: #F7CFCF;
+$cc-color-danger-light-bg: #FFEDED;
+$cc-color-danger-light-fg: #D03939;
+$cc-color-danger-light-hover: #F7CFCF;
 
 .btn-default-light {
-  background-color: @cc-color-default-light-bg !important;
-  color: @cc-color-default-light-fg !important;
+  background-color: $cc-color-default-light-bg !important;
+  color: $cc-color-default-light-fg !important;
   font-weight: bold;
 }
 
 .btn-default-light:hover {
-  background-color: @cc-color-default-light-hover !important;
+  background-color: $cc-color-default-light-hover !important;
 }
 
 .btn-default-light:focus {
-  outline-color: @cc-color-default-light-fg;
+  outline-color: $cc-color-default-light-fg;
 }
 
 .btn-success-light {
-  background-color: @cc-color-success-light-bg !important;
-  color: @cc-color-success-light-fg !important;
+  background-color: $cc-color-success-light-bg !important;
+  color: $cc-color-success-light-fg !important;
   font-weight: bold;
 }
 
 .btn-success-light:hover {
-  background-color: @cc-color-success-light-hover !important;
+  background-color: $cc-color-success-light-hover !important;
 }
 
 .btn-success-light:focus {
-  outline-color: @cc-color-success-light-fg;
+  outline-color: $cc-color-success-light-fg;
 }
 
 .btn-warning-light {
-  background-color: @cc-color-warning-light-bg !important;
-  color: @cc-color-warning-light-fg !important;
+  background-color: $cc-color-warning-light-bg !important;
```

Comment:
can these colors be revisited to use bootstrap5 color utilities and the base color updates (related to marketing's updates)? thanks!
https://www.commcarehq.org/styleguide/b5/atoms/colors/
https://getbootstrap.com/docs/5.3/customize/color/

for instance `$cc-color-warning-light-bg` should be `$warning-bg-subtle`, etc.

Reply:
@orangejenny $warning-subtle is referenced in the styleguide, but i think in this case it's best to use the one that's generated off of the theme colors rather than the hard coded ones in this file

Reply:
13ce811228e

I didn't redo the screenshots in the PR description since the changes are pretty subtle. Here's one example of button colors changing:
before
![Screenshot 2024-04-24 at 3 13 39 PM](https://github.com/dimagi/commcare-hq/assets/1486591/ae880f12-1e49-4376-928c-1b113dd40ddf)

after

![Screenshot 2024-04-24 at 3 13 33 PM](https://github.com/dimagi/commcare-hq/assets/1486591/63effb88-55aa-477b-804a-9f506437b480)